EnglishFilter/sections/specific.txt

```diff
@@ -258,6 +258,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 ||mypornstarbook.net/ads/
+amongusavatarmaker.com##h1 > div[align^="center"] > a[href] > img[border="0"]
```

Comment:
`||i.ibb.co^$domain=amongusavatarmaker.com`

Reply:
I don't trust it, they may change it.